inst/examples/colourInput/app.R

```diff
@@ -116,6 +117,39 @@ shinyApp(
 
     div(
       class = "section",
+      div(class = "title", "Allow alpha"),
```

Comment:
All of this code adds a ninth box for testing just allowAlpha. It may be better to roll allowAlpha into one of the others (put it with showBackground perhaps?) because 9 boxes is not the smoothest wrapping at a lot of window sizes.